aws/scripts/create_links_sinks.py

```diff
@@ -0,0 +1,168 @@
+        raise Exception('adding policy to sink')
+    
+    return sink['Arn']
+
+# Renders link-cfn-template.yaml as a string with user provided arguments
+def render_template(monitoring_account_id, sink_arn):
+    env = Environment(
+        loader=FileSystemLoader('./'),
+        autoescape=select_autoescape())
+    template = env.get_template('link-cfn-template.yaml')
+    return template.render(MonitoringAccountID=monitoring_account_id, SinkARN=sink_arn)
+
+# Creates a stackset and stack instances that create links to the created sink
+def create_stackset(region, profile, sink_arn, organization_unit, stack_set_name, excluded_accounts):
+    # Get CFN Client
+    client = get_client(profile, 'cloudformation', region)
+
+    # Get account ID from caller identity
+    try:
+        account_id = get_client(profile, 'sts', region).get_caller_identity()['Account']
+    except Exception:
+        raise Exception('get account id')
+
+    # Execute template
+    try:
+        rendered_template = render_template(monitoring_account_id=account_id, sink_arn=sink_arn)
+    except Exception:
+        raise('rendering template')
+    
+    # Create stackset
+    try:
+        stack_set = client.create_stack_set(
+            StackSetName=stack_set_name,
+            Description="Source Account Links To Enable Cross-Account Monitoring",
+            TemplateBody=rendered_template,
+            PermissionModel='SERVICE_MANAGED',
+            AutoDeployment={
+                'Enabled': True,
+                'RetainStacksOnAccountRemoval': False
+            }
+        )
+    except Exception:
+        raise('creating stack set')
+
+    # Set DeploymentTargets
+    deployment_targets={
+        'OrganizationalUnitIds': [
+            organization_unit,
+        ],
+        'AccountFilterType': 'NONE'
+    }
+    if excluded_accounts is not None:
+        deployment_targets['Accounts'] = excluded_accounts
+        deployment_targets['AccountFilterType'] = 'DIFFERENCE'
+
+    # Create stack instances
+    client.create_stack_instances(
+        StackSetName=stack_set['StackSetId'],
+        DeploymentTargets=deployment_targets,
+        Regions=[region],
+        OperationPreferences={
+            'RegionConcurrencyType': 'SEQUENTIAL',
+            'MaxConcurrentCount': 1,
+        }
+    )
+    return
+
+def monitoring_onboarding():
+    # Fetch/validate CLI args
+    args = parser.parse_args()
+    arg_dict = vars(args)
+    if arg_dict['organization'] is not None:
+        organization = arg_dict['organization']
+    else:
+        raise Exception('--organization-id is required')
+
+    if arg_dict['organization_unit'] is not None:
+        organization_unit = arg_dict['organization_unit']
+    else:
+        raise Exception('--organization-unit is required')
+
+    if arg_dict['regions'] is not None:
+        regions = arg_dict['regions'].split(',')
+    else:
+        regions = aws_default_regions
+    
+    sink_name = 'Centralized-Monitoring'
```

Comment:
In the `argparse` definitions, set the key `default` with any default values. Applies throughout